### ISSUE TITLE
v2 of the API requires an update to EIAdata

### ISSUE THREAD

**Issue by Matt-Brigida** [Matt-Brigida/EIAdata#19]
Fix according to:

https://www.eia.gov/opendata/documentation.php

**Comment by sawyervillers** [comment]
Matt-Any thoughts on whether a fix is imminent?  I'm not a programmer, so all I can offer is encouragement for a very useful package that's part of my daily workflow.

**Comment by Matt-Brigida** [comment]
@sawyervillers the [documentation](https://www.eia.gov/opendata/documentation.php) says I should be able to request `xml`, however the API presently ignores the request and returns `json`.  I have heard there may be some issues on the EIA's end.  I'll wait a day or so and try again.

If it returns `xml` the fix will be quick.  If it requires a rewrite to handle `json` data the fix will take some time. Wish I could be more exact.

**Comment by Matt-Brigida** [comment]
I added a fix in the `v2_fix` branch.  When testing I realized in the new API the data is sometimes returned under `value` and other times under `price`.  Additionally, my fix would only work if you knew the series ID, and the EIA removed the way to find these from their website. 

`EIAdata` would therefore require a significant rewrite which I don't have the time to do.  I have requested `CRAN` remove the `EIAdata` package and will archive this repo.  

A big thanks to all who added code and helped maintain `EIAdata` over the years.
